[Jekins Keycloak CI] - RH-SSO EAP adapters remote saml tests

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug


```
....
2024-12-01 21:59:14  [ERROR]   RemoteSAMLFilterAdapterTest.testUserAttributeStatementMapperGroupsNoAggregate » WebDriver
2024-12-01 21:59:14  [ERROR]   RemoteSAMLFilterAdapterTest.testUserAttributeStatementMapperUserGroupsAggregate » WebDriver
2024-12-01 21:59:14  [ERROR]   RemoteSAMLFilterAdapterTest.testUserAttributeStatementMapperUserGroupsNoAggregate » WebDriver
2024-12-01 21:59:14  [ERROR]   RemoteSAMLFilterAdapterTest.unauthorizedSSOTest » WebDriver com.gargoylesoftwa...
2024-12-01 21:59:14  [INFO] 
2024-12-01 21:59:14  [ERROR] Tests run: 170, Failures: 8, Errors: 96, Skipped: 5
2024-12-01 21:59:14  [INFO] 
2024-12-01 21:59:14  [INFO] ------------------------------------------------------------------------
2024-12-01 21:59:14  [INFO] BUILD FAILURE
```

`Caused by: com.gargoylesoftware.htmlunit.ScriptException: TypeError: Cannot find function requestSubmit in object [object HTMLFormElement]. (script in [http://127.0.0.1]`


https://keycloak-jenkins/view/Keycloak/job/universal-test-pipeline-adapters/2701/execution/node/987/log/

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

--- a/themes/src/main/resources/theme/base/login/saml-post-form.ftl
+++ b/themes/src/main/resources/theme/base/login/saml-post-form.ftl
@@ -3,7 +3,7 @@
     <#if section = "header">
         ${msg("saml.post-form.title")}
     <#elseif section = "form">
-        <script>window.onload = function() {document.forms[0].requestSubmit()};</script>
+        <script>window.onload = function() {document.forms[0].submit()};</script>
         <p>${msg("saml.post-form.message")}</p>
         <form name="saml-post-binding" method="post" action="${samlPost.url}">
             <#if samlPost.SAMLRequest??>

